environment/helm_chart_connections.go

```diff
@@ -22,6 +24,24 @@ const (
 	HTTPS
 )
 
+// FindRemoteURLs returns URLs for any chart by prefix/portName/connectionType
+func FindRemoteURLs(e *Environment, prefix string, portName string, connectionType Protocol) func(e *Environment) ([]*url.URL, error) {
```

Comment:
Got an example of how this is used? Also the func says remote but it gets local URLs